src/main/java/eiteam/esteemedinnovation/commons/util/ReflectionHelper.java

```diff
@@ -65,7 +65,7 @@ public static Method getMethod(String methodName, String obfName, Class clazz, C
                     builder.append(method1.getName());
                     builderHasAMethod = true;
                 }
-                FMLLog.warning("[EI] Unable to find " + methodName + " method in " + clazz.getName() +
+                EsteemedInnovation.logger.warn("[EI] Unable to find " + methodName + " method in " + clazz.getName() +
```

Comment:
`[EI]` should be removed from this, because it says `[esteemedinnovation]` in each log entry